config/src/main/java/org/springframework/security/config/annotation/web/configurers/saml2/Saml2LoginConfigurer.java

```diff
@@ -221,6 +221,10 @@ public void init(B http) throws Exception {
 				// Setup auto-redirect to provider login page
 				// when only 1 IDP is configured
 				this.updateAuthenticationDefaults();
+				Saml2LogoutConfigurer<B> logoutConfigurer = http.getConfigurer(Saml2LogoutConfigurer.class);
```

Comment:
I'll look into this. Note that this was in place primarly in order to follow how it's done in `AbstractAuthenticationFilterConfigurer`, though I have otherwise no specific reason to support the current arrangement.